docs/tools/swift.pod

```diff
@@ -1,30 +1,106 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language by Apple Inc.
 
 =head1 SYNOPSIS
 
-B<swift> [-help] [I<input-filename> [I<program-arguments>]]
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
```

Comment:
Maybe also add:

To execute a swift program:

    swift program.swift -- <args>